boot/kube-apiserver.sh

```diff
@@ -19,6 +19,8 @@ exec $(dirname $0)/nsenter.sh kube-apiserver \
 	--service-account-signing-key-file=$XDG_CONFIG_HOME/usernetes/master/service-account-key.pem \
 	--advertise-address=$(cat $XDG_RUNTIME_DIR/usernetes/parent_ip) \
 	--allow-privileged \
+	--encryption-provider-config=$XDG_CONFIG_HOME/usernetes/master/secrets-encryption.yml \
+	--authorization-mode=Node,RBAC \
```

Comment:
Could you split and squash the commits so that one commit has one topic?

Probably, this PR should have the following three commits:
1. RBAC
2. Secrets encryption
3. Calico

Reply:
How should I handle `install.sh` as it has edits for RBAC, Calico and secrets encryption ?

Reply:
Just make commit 3 depend on commit 1 and 2?